cli/cppcheckexecutor.cpp

```diff
@@ -266,7 +281,10 @@ int CppCheckExecutor::check_internal(CppCheck& cppcheck)
         mLatestProgressOutputTime = std::time(nullptr);
 
     if (!settings.outputFile.empty()) {
-        mErrorOutput = new std::ofstream(settings.outputFile);
+        mErrorOutput = new std::ofstream();
+        mErrorOutput->exceptions(std::ios_base::failbit | std::ios_base::badbit);
+        mErrorOutput->open(settings.outputFile);
+        assert(mErrorOutput->is_open()); // If we ever disable exceptions, catch issues ASAP
```

Comment:
If we ever disable exception nothing will work at all so this is unnecessary. It will also only trigger in a debug build.